build-bin/docker-compose.test.yml

```diff
@@ -31,14 +33,14 @@ services:
       backend:
         condition: service_healthy
       zipkin:
-        condition: service_healthy
+        condition: service_started
   backend:
     container_name: backend
     image: ${DOCKER_IMAGE}
     entrypoint: start-backend
     depends_on:
       zipkin:
-        condition: service_healthy
+        condition: service_started
```

Comment:
this is more a documentation style dep. we could leave it out entirely as iiuc services can see all the hosts regardless of depends_on